docs/setup.asciidoc

```diff
@@ -80,7 +103,7 @@ You can also configure the agent using `web.config` as described at <<configurat
 
 [float]
 [[setup-ef6]]
-=== Entity Framework 6
+==== Entity Framework 6
 
 You can enable auto instrumentation for Entity Framework 6 by referencing the `Elastic.Apm.EntityFramework6` package
 and including the `Ef6Interceptor` interceptor in your application's `web.config`:
```

Comment:
I've redesigned the layout of the **Setup** page to *hopefully* be easier to digest. Please take a look and compare the changes: **NEW LAYOUT** /vs/ **OLD LAYOUT**

Reply:
Awesome, I love it!